src/actions/acquire_single_freq_fft.py

```diff
@@ -277,30 +318,25 @@ def build_sigmf_md(self, task_id, data):
         time_domain_avg_power += (
             10 * np.log10(1 / (2 * 50)) + 30
         )  # Convert log(V^2) to dBm
-        sensor_overload = (
-            time_domain_avg_power
-            > self.sdr.radio.sensor_calibration_data["1db_compression_sensor"]
-        )
+        sensor_overload = False
+        if self.sdr.radio.sensor_calibration_data["1db_compression_sensor"]:
```

Comment:
If `1db_compression_sensor` was exactly 0 (e.g. `0.0`), would this evaluate to `False` and not check compression?

Currently, a this value is set to 100dBm (unrealistic for these sensors) is it is not in the calibration file, but it may be better to leave this unset and check for a key here (or check to see if `1db_compression_sensor` > 99).

Reply:
Changed to check `is not None`